app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -488,4 +488,6 @@
   <string name="faq_answer_9">&lt;p&gt;Se você não consegue encontrar sua pergunta ou gostaria de relatar um problema, entre em contato conosco em admin@oppia.org.&lt;/p&gt;</string>
   <string name="profile_edit_fragment_test_activity_label">Atividade de Teste de Fragmento de Edição de Perfil</string>
   <string name="administrator_controls_fragment_test_activity_label">Controle Administrativo da Atividade de Teste de Fragmento</string>
+  <string name="select_a_topic_to_start">Selecione um Tópico para começar</string>
+  <string name="profile_edit_activity_title">"Editar perfil "</string>
```

Comment:
@DeonWaju Please remove space before last quote `<string name="profile_edit_activity_title">"Editar perfil "</string>` to `<string name="profile_edit_activity_title">"Editar perfil"</string>`